neo4j-ogm-docs/src/main/asciidoc/reference/nativetypes.adoc

```diff
@@ -0,0 +1,183 @@
+[[reference:native-property-types]]
+= Native property types
+
+Neo4j distinguishes between property, structural and composite types.
+While you can map attributes of Neo4j-OGM entities very easy to composite types
+(please read <<reference:type-conversion:custom:using-composite-types, the example using a custom converter for composite types>> for more information),
```

Comment:
Perhaps this part in brackets could be moved to a non-bracketed sentence after the current sentence is complete.

Reply:
👍